index.js

```diff
@@ -22,8 +22,9 @@ var amortizationCalc = function(amount, rate, totalTerm, amortizeTerm, principal
       monthlyIntPaid,
       monthlyPrincPaid,
       summedAmortize = {},
-      montlyPrincipalPayment = 0,
-      principalPayment = (!!principalPayment) ? parseInt(principalPayment,10) : 0;
+      principalPayment = (!!principalPayment) ? parseInt(principalPayment,10) : 0,
+      principalBreakingTerm = 0,
+      montlyPrincipalPayment = 0;
```

Comment:
It looks like this is a typo and this should be `monthlyPrincipalPayment`

Reply:
This came from a master merge, do you want to change it in master?

https://github.com/cfpb/amortize/pull/14/files#diff-168726dbe96b3ce427e7fedce31bb0bcR25

Reply:
Good call @avantassel, I'll fix that in master.

Reply:
Ahh yes! Thanks @avantassel and @contolini